code/modules/power/apc.dm

```diff
@@ -130,6 +131,9 @@
 	req_access = list(ACCESS_SYNDICATE)
 	report_power_alarm = FALSE
 
+/obj/machinery/power/apc/constructed // APCS the crew builds
```

Comment:
This subtype isnt needed. Look at line 167. Just assign the var inside that if block.

Reply:
Done, thanks for the help